apps/configurator/lib/configurator_web/controllers/character_controller.ex

```diff
@@ -0,0 +1,75 @@
+defmodule ConfiguratorWeb.CharacterController do
+  use ConfiguratorWeb, :controller
+
+  alias Configurator.Configuration
+  alias Configurator.Configuration.Character
+  alias ConfiguratorWeb.UtilsAPI
+
+  def index(conn, _params) do
+    characters = Configuration.list_characters()
+    render(conn, :index, characters: characters)
+  end
+
+  def new(conn, _params) do
+    changeset = Configuration.change_character(%Character{})
+    render(conn, :new, changeset: changeset)
+  end
+
+  def create(conn, %{"character" => character_params}) do
+    # TODO this should be removed once we have the skills relationship
```

Comment:
Same here, let's add the its issue link.

Reply:
Done